### PR TITLE
Correct documents and optimise set_version.py

### PR DESCRIPTION
**Pull request by tjone270**
Hi there,

I've made some improvements to your documentation and to one of your Python scripts.

Thanks,
Thomas Jones

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -24,9 +24,12 @@
 
 ### Get source
 
-1. Install git
-2. Get source code:
-
+1. Install git: 
+    ```
+    sudo apt install git
```
Comment (uglide) [inline docs/install.md]: Please remove this change from PR. This section describes steps for all platforms, not just for Debian/Ubuntu flavors with `apt-get`.

**Review by uglide — CHANGES_REQUESTED** [review]

**Comment by uglide** [comment]
Please also rebase your changes on latest head of 2019 branch.

Thanks for your PR.

File: docs/install.md (hunk 1 above)
Comment (tjone270) [reply docs/install.md]: This is now done.

**Comment by mutoubed** [comment]
i want to try

**Comment by tjone270** [comment]
@uglide I've made the changes you've requested.

File: docs/install.md [hunk 2]
```diff
@@ -27,9 +27,9 @@
 
 ### Get source
 
-1. Install git
-2. Get source code:
-
+1. Install git using the instructions here: https://git-scm.com/download/linux
```
Comment (uglide) [inline docs/install.md]: Please remove linux specific link, this link isn't helpful for users on macOS and Windows

**Comment by uglide** [comment]
@tjone270 Your PR still needs some changes (see my comment) and should be rebased on latest head of 2019 branch.

File: docs/install.md (hunk 2 above)
Comment (tjone270) [reply docs/install.md]: Completed.

**Comment by tjone270** [comment]
@uglide I've made the changes you've requested and the repository is now rebased on the head of the 2019 branch.

**Comment by uglide** [comment]
@tjone270 you have messed up a bit with rebase, so I rebased your changes and merged as https://github.com/uglide/RedisDesktopManager/pull/4706

Thanks for your contribution!

**Comment by tjone270** [comment]
@uglide sorry about that, thanks very much!

**Pull request closed without merging**